lib/alertStateMachine.js

```diff
@@ -15,7 +15,11 @@ class AlertStateMachine {
     switch (currentAlertState) {
       case CHATBOT_STATE.STARTED:
       case CHATBOT_STATE.WAITING_FOR_REPLY:
-        nextAlertState = CHATBOT_STATE.WAITING_FOR_CATEGORY
+        if (messageText.trim().toLowerCase() === 'reset') {
+          nextAlertState = CHATBOT_STATE.RESET
+        } else {
+          nextAlertState = CHATBOT_STATE.WAITING_FOR_CATEGORY
+        }
         break
 
       case CHATBOT_STATE.WAITING_FOR_CATEGORY:
```

Comment:
What do you think about adding `case CHATBOT_STATE.RESET` down by `case CHATBOT_STATE.COMPLETED`? I know that it will be caught by the `default` anyways, but this will make it clear that that is the only intended thing to happen if anyone does send a text message for a session that is in the `RESET` state.

Reply:
Ah, missed this; good catch.